File: kyonggiyo-core/src/main/java/kyonggiyo/domain/image/exception/ImageException.java
package kyonggiyo.domain.image.exception;

import kyonggiyo.common.exception.BusinessException;
import kyonggiyo.common.exception.ErrorCode;

public class ImageException extends BusinessException {

    public ImageException(ErrorCode errorCode) {
        super(errorCode);
    }

    public ImageException(ErrorCode errorCode, String loggingMessage) {
        super(errorCode, loggingMessage);
    }

}

File: kyonggiyo-core/src/main/java/kyonggiyo/application/image/domain/exception/ImageException.kt
package kyonggiyo.application.image.domain.exception

import kyonggiyo.common.exception.BusinessException
import kyonggiyo.common.exception.ErrorCode

class ImageException : BusinessException {
    constructor(errorCode: ErrorCode?) : super(errorCode!!)
    constructor(errorCode: ErrorCode?, loggingMessage: String?) : super(errorCode!!, loggingMessage!!)
}
